src/components/canvas-tools/tool-tile.tsx

```diff
@@ -65,6 +65,7 @@ export class ToolTileComponent extends BaseComponent<IProps, {}> {
     }
     return (
       <div className={`tool-tile${selectedClass}`}
+          data-tool-id={model.id}
```

Comment:
What are these `data` fields for?

Reply:
That's a good question as it turns out. I added them originally because I thought I was going to have to find the newly created row/tile by ID to determine which one to scroll into view. Eventually, I decided that it made sense to scroll any new row/tile into view, so didn't end up using them. I think they're useful for debugging, however, and I'm fairly certain we'll need them at some point so I decided to leave them in.